### DISCUSSION TITLE
Roadmap for Quant Finance

### DISCUSSION THREAD

**Discussion by mAndar048** [OpenBB-finance/OpenBB#5769] [Q&A]
I made a roadmap for the quant finance field, I did some research, but I still don't know what is needed to enter this field.
https://roadmap.sh/r?id=65389cc9035e8d1be7367e20   
Please Help me with New topics and Resources .

**Comment by tilakpatel22** [comment]
@mAndar048 Hello, I would like to help/contribute to your quant finance roadmap, are you still available?

**Reply by mAndar048** [reply]
Hi , i have turned on The sharing Setting, You can View It and for contribution, U can Mail me At my Personal mail, and we can talk about it ..

**Comment by tilakpatel22** [comment]
@mAndar048 can you include your mail here i can't find

**Reply by mAndar048** [reply]
mandarjoshi9130@gmail.com

**Comment by Harshvasava307** [comment]
Think of quant finance as 3 skills stacked together:
Math brain + Coding hands + Market sense

Stage 1: Build Your Brain Tools
Stage 2: Learn to Speak to Computers
Stage 3: Understand Money & Markets

i know only 3 Stages as of i am already into it

**Reply by CY5** [reply]
Clear roadmap, that’s usually how stage 3 thinking shows up.
